pages/common/xmlstarlet.md

```diff
@@ -0,0 +1,33 @@
+# xmlstarlet
+
+> A commandline XML/XSLT toolkit.
+> Note: You will likely need to know XPath: <https://developer.mozilla.org/en-US/docs/Web/XPath>.
+> More information: <https://xmlstar.sourceforge.net/docs.php>.
+
+- Format an XML document and print to stdout:
+
+`xmlstarlet format {{path/to/file.xml}}`
+
+- XML document can also be piped from stdin:
+
+`{{cat path/to/file.xml}} | xmlstarlet format`
+
+- Print all nodes that match a given XPath:
+
+`xmlstarlet select --template --copy-of {{xpath}} {{file.xml}}`
+
+- Insert an attribute to all matching nodes, and print to stdout (source file is unchanged):
+
+`xmlstarlet edit --insert {{xpath}} --type attr --name {{attribute_name}} --value {{attribute_value}} {{file.xml}}`
+
+- Rename some nodes, delete some nodes, and print to stdout (source file is unchanged):
+
+`xmlstarlet edit --rename {{xpath_to_rename}} --value {{new_name}} --delete {{xpath_to_delete}} {{file.xml}}`
```

Comment:
I would suggest splitting this into 2 examples for readability. Besides, I don't think people rename and delete nodes at the same time.

Reply:
Will do.